content/docs/news/release_announcements/keptn-110/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Keptn 1.1.0
+weight: 63
+---
+
+## [1.1.0](https://github.com/keptn/keptn/compare/1.0.0...1.1.0) (2023-01-18)
+
+
+### Features
+
+* **bridge:** Read secrets from volume mount instead from env var ([#9335](https://github.com/keptn/keptn/issues/9335)) ([94c60e5](https://github.com/keptn/keptn/commit/94c60e55a1240660ea07e1f35e8afdf809b4d4a0))
+* **bridge:** Sort registration items by name ([#9344](https://github.com/keptn/keptn/issues/9344)) ([6dcf6f6](https://github.com/keptn/keptn/commit/6dcf6f686c6ea211bcdae163c52e9fa6fc79f631))
+* **resource-service:** reduce git fetch operations ([#9410](https://github.com/keptn/keptn/issues/9410)) ([4a2b091](https://github.com/keptn/keptn/commit/4a2b091aac9006cfc0343db8eb3cf215f6557feb))
+
+
+### Bug Fixes
+
+* **cli:** display incompatible upgrade possibilities even if no minor upgrade is available ([#9381](https://github.com/keptn/keptn/issues/9381)) ([7d66999](https://github.com/keptn/keptn/commit/7d6699960700925cde5bfbefd1f8fb722b5b6dd2))
+* Fix generation of SBOMs during releases of Keptn ([#9445](https://github.com/keptn/keptn/issues/9445)) ([5f01217](https://github.com/keptn/keptn/commit/5f01217cac3a8add79100506d7718bb40d5e7305))
+* Fix some shellcheck issues in pipeline related shell scripts ([#9322](https://github.com/keptn/keptn/issues/9322)) ([676a2d0](https://github.com/keptn/keptn/commit/676a2d07ede016ca1ba2885d157d8d483b8e960a))
```

Comment:
```suggestion
* Fix some shellcheck issues in pipeline-related shell scripts ([#9322](https://github.com/keptn/keptn/issues/9322)) ([676a2d0](https://github.com/keptn/keptn/commit/676a2d07ede016ca1ba2885d157d8d483b8e960a))
```
I think this is what you mean.  Alternatively, you might mean "in pipelines related to shell scripts".